thoth-tech/documentation
# Splashkit: WSL installation and FAQs

# Description

_I have added an additional method of installing SplashKit on Windows through WSL (Windows Subsystem for Linux). This change also includes troubleshooting FAQs for both Windows and Mac users during the SplashKit installation process.._

Fixes # (issue)

## Type of change

_Please delete options that are not relevant._


# How Has This Been Tested?

I have tested the installation method on a WSL environment and verified that it works correctly. To reproduce the test:
1.  Set up a Windows machine with WSL installed.
1.  Follow the installation guide provided in the Pull Request.
1.  Verify that SplashKit is successfully installed on the Windows machine.

## Testing Checklist:

- [x] Tested on WSL environment.

# Checklist:

- [x] I have commented on my code in hard-to-understand areas
- [x] I have requested a review from ... on the Pull Request
